src/testing/AddTestToIdentifierPage.test.tsx

```diff
@@ -14,77 +15,140 @@ jest.mock('../common/useConfig');
 describe(AddTestToIdentifierPage, () => {
   beforeEach(() => {
     mockConfigForEstonianIdMethodAndEnglish();
-    mockToken('some-token');
+    mockAuthentication();
   });
 
   it('focuses identifier input', async () => {
-    const { getByLabelText } = renderWrapped(<AddTestToIdentifierPage />);
+    mockHttp().get('/api/v1/test-types').reply(200, [aTestType()]);
 
-    const identifierInput = getByLabelText(/identification code/i);
+    renderWrapped(<AddTestToIdentifierPage />);
 
+    const identifierInput = screen.getByLabelText(/identification code/i);
     expect(identifierInput).toHaveFocus();
   });
 
-  it('creates a user for the authentication method from config and the filled identifier', async () => {
-    const authenticationDetails = { method: 'ESTONIAN_ID', identifier: '39210030814' };
-    const scope = nock(/./)
-      .post('/api/v1/users', { authenticationDetails })
-      .matchHeader('Authorization', 'Bearer some-token')
-      .reply(201, { id: 'some-id', authenticationDetails });
+  it('creates a user for the authentication method from config and the filled identifier and adds a test for that user', async () => {
```

Comment:
Generally a bit hard to follow as the test does a lot of things.

Reply:
Indeed — I'll try to split some of the logic out!

Reply:
Split validation testing, added helper functions for readability, and moved more common setup, including the render, to `beforeEach` 👍